ce/api/multistats.py

```diff
@@ -5,7 +5,7 @@
 from ce.api.util import search_for_unique_ids
 
 def multistats(sesh, ensemble_name='ce_files', model='', emission='', time=0,
-               area=None, variable='', timescale=''):
+               area=None, variable='', timescale='', cell_method=''):
```

Comment:
Maybe we should include `mean` as the default value for `cell_method`? That would be backward compatible, right?